integration-tests/oidc-code-flow/src/main/resources/application.properties

```diff
@@ -44,8 +44,24 @@ quarkus.oidc.tenant-3.authentication.redirect-path=/some/other/path
 quarkus.oidc.tenant-3.authentication.restore-path-after-redirect=false
 quarkus.oidc.tenant-3.application-type=web-app
 
+quarkus.oidc.tenant-logout.auth-server-url=${keycloak.url}/realms/quarkus
+quarkus.oidc.tenant-logout.client-id=quarkus-app
```

Comment:
`quarkus-app-logout` as a client id would help with the proposed simplifications

Reply:
It won't work. The changes I need are at the realm. I would need to create a new realm only for this test.

Reply:
@pedroigor so what is the problem with creating a new realm as well ?